modules/apigee_edge_debug/src/HttpClientMiddleware/ApiClientProfiler.php

```diff
@@ -141,6 +141,18 @@ public function __invoke() {
             }
             $context['error'] = $error;
           }
+
+          // If devel kint module is enabled and the user has devel kint permission.
+          $user = \Drupal::currentUser();
```

Comment:
\Drupal:: should be avoided because it makes unit testing impossible. Dependency injection should be used instead.

Reply:
Hi @mxr576 
Thanks for the suggestions! We will implement the changes requested and submit a new PR.